samples/csharp/end-to-end-apps/Model-Explainability/TaxiFarePrediction.Explainability/TaxiFarePrediction.Explainability.csproj

```diff
@@ -2,8 +2,6 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\..\packages\Microsoft.ML.1.4.0\build\netstandard2.0\Microsoft.ML.props" Condition="Exists('..\..\..\packages\Microsoft.ML.1.4.0\build\netstandard2.0\Microsoft.ML.props')" />
   <Import Project="..\..\..\packages\Microsoft.ML.CpuMath.1.4.0\build\netstandard2.0\Microsoft.ML.CpuMath.props" Condition="Exists('..\..\..\packages\Microsoft.ML.CpuMath.1.4.0\build\netstandard2.0\Microsoft.ML.CpuMath.props')" />
-  <Import Project="..\packages\Microsoft.ML.1.3.1\build\netstandard2.0\Microsoft.ML.props" Condition="Exists('..\packages\Microsoft.ML.1.3.1\build\netstandard2.0\Microsoft.ML.props')" />
```

Comment:
where you able to run this sample after changing this, @harishsk ?

Reply:
After doing these changes on this .csproj, I'm still unable to run this desktop app 😕